pages/installation/operating_system_packages.rst

```diff
@@ -23,21 +23,21 @@ Make sure to install and configure the following software before installing and
 
 * Java (>= 8)
 * MongoDB (>= 2.4)
-* Elasticsearch (>= 2.x)
+* Elasticsearch (>= 5)
```

Comment:
The system requirements are *still* Elasticsearch 2.x or later (I think 2.1.0 to be precise, but 2.x will do in the docs).